client/src/Annotator/Info/useFetch.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Fetching dataset data
+ */
+import { useState, useEffect } from 'react';
+
+import { Dataset, Category } from '../../common/types';
+import { Maybe } from '../annotator.types';
+
+import * as AnnotatorApi from '../annotator.api';
+import useGlobalContext from '../../common/hooks/useGlobalContext';
+import { addProcess, removeProcess } from '../../common/utils/globalActions';
+
+const useFetch = (imageId: number) => {
```

Comment:
useFetch isn't a proper name for this hook. It is named like a hook that would use standard `fetch` functionality from a browser to perform XHR request.